assets/app/styles/_components.less

```diff
@@ -52,6 +52,9 @@
       + span {
         margin-left: 5px;
       }
+      .component-status {
```

Comment:
we'll also probably use this when we inline the service name with the component label... maybe call it `component-label-details`?